metamer/ftest/src/test/java/org/richfaces/tests/metamer/ftest/extension/arquillian/JBossPatchInstaller.java

```diff
@@ -136,23 +77,14 @@ public void applyPatchWhenNeeded(@Observes BeforeStart event) {
             logInfoMarked("No patch to apply.");
         } else {
             if (new File(pathToPatch).exists()) {
-                try {
-                    applyPatch(pathToPatch);
-                } catch (CommandLineException e) {
-                    logError(e.toString());
-                    logError("Was not able to apply the patch. Exiting.");
-                    System.exit(1);
-                }
+                applyPatch(pathToPatch);
             } else {
                 logError(format("The patch at <{0}> does not exist. Exiting.", pathToPatch));
                 System.exit(1);
```

Comment:
Originally there was also called `System.exit` when `applyPatch` failed, we should either in both cases throw exception or do `System.exit`

NOTE, I've noticed that `applyPatch` actually does also System.exit, so there is no inconsistency. Still I would prefer rather having thrown exception. Exception is more informative than exit code.